test-butler-app-core/build.gradle

```diff
@@ -27,33 +23,10 @@ android {
         textOutput 'stdout'
         abortOnError true
         warningsAsErrors true
-        disable 'OldTargetApi', 'GradleDependency', 'PrivateApi', 'WakelockTimeout'
-    }
-}
-
-task sourcesJar(type: Jar) {
-    classifier = 'sources'
-    from android.sourceSets.main.java.srcDirs
-}
-
-task javadoc(type: Javadoc) {
-    source = android.sourceSets.main.java.srcDirs
-    android.libraryVariants.matching { variant -> variant.name == 'debug'}.all { variant ->
-        classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-        classpath += variant.getCompileClasspath(null)
+        disable 'OldTargetApi', 'GradleDependency', 'PrivateApi', 'WakelockTimeout', 'DiscouragedPrivateApi'
```

Comment:
Are these new lint checks introduced by AGP bump? Just to confirm

Reply:
Yep, that's correct.